tests/Integration/GameSessionEntitiesTest.php

```diff
@@ -82,7 +83,7 @@ public function testLayerEntity(): void
         $this->start();
         $layer = new Layer();
         $layer->setLayerName('test');
-        $layer->setLayerGeotype('raster');
+        $layer->setLayerGeotype('polygon');
```

Comment:
via enum?

Reply:
will have to tackle this some other time - need to move on to other priority tasks